### PR TITLE
The link directive should be repeatable

### PR DESCRIPTION
**Pull request by cschiewek**
The `link` directive should be repeatable.  See https://www.apollographql.com/docs/federation/federated-types/federated-directives/#the-link-directive

cc: @kzlsakal

### CODE REVIEW

**Comment by cschiewek** [comment]
Would love to see a 0.4.1 release if/when this is merged.

**Review by kzlsakal — CHANGES_REQUESTED** [review]
Thanks @cschiewek! Would you be able to add a test that verifies the repeatable behaviour?

**Comment by cschiewek** [comment]
@kzlsakal Would `Absinthe.Federation.NotationTest` be the appropriate module for that test? I don't see any other test that explicitly checking any of the federation directives.

File: test/absinthe/federation/notation_test.exs [hunk 1]
```diff
@@ -95,5 +95,28 @@ defmodule Absinthe.Federation.NotationTest do
...
+            url: "https://myspecs.example.org/myDirective/v1.0",
+            import: ["@myDirective"]
+        end
+
+        query do
+          field :hello, :string
+        end
+      end
+
+      assert Absinthe.Schema.to_sdl(MultipleLinkSchema)
```
Comment (kzlsakal) [inline test/absinthe/federation/notation_test.exs:119]: Do you mind checking to ensure `@key`, `@tag`, and `@myDirective` are all printed on the output of this `to_sdl/1` call? See the test right above for a similar assertion.

**Review by kzlsakal — APPROVED** [review]
Thank you very much @cschiewek!

**Comment by cschiewek** [comment]
Thank you for the quick release @kzlsakal!

**Pull request merged**